src/common/components/hosts/hardwareInfo.ts

```diff
@@ -21,8 +21,9 @@ export type SimpleHardwareInfo = {
 export const getMemoryCapacity = (inventory: Inventory): number =>
   inventory.memory?.usableBytes || 0;
 export const getDiskCapacity = (inventory: Inventory): number =>
-  inventory.disks?.reduce((diskSize: number, disk: Disk) => diskSize + (disk.sizeBytes || 0), 0) ||
-  0;
+  inventory.disks
+    ?.filter((disk) => disk.driveType !== 'ODD')
```

Comment:
nit: A new constant would be better here. That way, we a future maintainer can find that faster and reliably.

Reply:
Done, 10x